stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -1178,7 +1178,7 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     /// - parameter url: The `URL` to read.
     /// - parameter options: Options for the read operation. Default value is `[]`.
     /// - throws: An error in the Cocoa domain, if `url` cannot be read.
-    public init(contentsOf url: URL, options: Data.ReadingOptions = []) throws {
+    public init(contentsOf url: __shared URL, options: __shared Data.ReadingOptions = []) throws {
```

Comment:
Data.ReadingOptions is an ObjC enum, right? So it shouldn't matter what ownership you use.

Reply:
Sure — some of the changes here were mechanical, so I can pull this one out. I'm surprised the compiler didn't produce a warning/error in this case.